clarifai.go

```diff
@@ -37,7 +37,7 @@ func NewClarifaiClient() *ClarifaiClient {
 }
 
 // Check whether the client still has access rights to Clarifai
-func (client *ClarifaiClient) isAccess() bool {
+func isCLFAccessible(client *ClarifaiClient) bool {
```

Comment:
For sure, both work. Previously, clairfai.go is like a class definition, all functions related with clarifai are defined as ClarifaiClient methods. And we can instantiate the ClarifaiClient struct (class) (it would be a singleton in our case) in the main.go, and we can access all the methods through this ClairfaiClient. It's nice and clear. An alternative approach is moving those clarifai methods out of box as normal functions (which can be accessible anywhere in the same package), and delare the ClarifaiClient struct in the main.go as a global variable. The reasons that I think the later is better are: 
1. golang is not primarily OO language, and I didn't see benefits of using object-oriented feature in Go to make the functions as methods here. ClarifaiClient is just a struct that holds the info of token and expiration_time, we just need one object of it instead of many different instances of it. 
2. Easy to work with other frameworks like Gin. For example, if we want to register a handler function to a specific route in Gin (the handler function and the ClairfaiClient are defined in different files) then we have to pass the ClarifaiClient to the handler function, which is difficult because typical ginHandler only takes gin.Context as argument